scss/_progress.scss

```diff
@@ -26,80 +26,49 @@
   // Reset the default appearance
   appearance: none;
   // Set overall border radius
-  @include border-radius($border-radius);
+  @include border-radius($progress-border-radius);
 }
 .progress[value]::-ms-fill {
-  background-color: $progress-bar-color;
   // Remove right-hand border of value bar from IE10+/Edge
   border: 0;
 }
 .progress[value]::-webkit-progress-bar {
   background-color: $progress-bg;
-  @include border-radius($border-radius);
+  @include border-radius($progress-border-radius);
   @include box-shadow(inset 0 .1rem .1rem rgba(0,0,0,.1));
 }
 .progress[value]::-webkit-progress-value {
-  background-color: $progress-bar-color;
-  border-top-left-radius: $border-radius;
-  border-bottom-left-radius: $border-radius;
+  border-top-left-radius: $progress-border-radius;
+  border-bottom-left-radius: $progress-border-radius;
 }
 .progress[value="100"]::-webkit-progress-value {
-  border-top-right-radius: $border-radius;
-  border-bottom-right-radius: $border-radius;
+  border-top-right-radius: $progress-border-radius;
```

Comment:
Properties should be ordered border-bottom-right-radius, border-top-right-radius